### PR TITLE
Graphql-ify SimilarityScore

### PR DESCRIPTION
**Pull request by shichongrui**
refs: MBL-12810
affects: Teacher
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1007133&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|4
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|4

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Please add tests for these files: [rn/Teacher/src/modules/graphql-speed-grader/components/SimilarityScore.js](https://github.com/instructure/canvas-ios/blob/gql-similarity-scorern/Teacher/src/modules/graphql-speed-grader/components/SimilarityScore.js)
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Teacher

[MBL-12810](https://instructure.atlassian.net/browse/MBL-12810)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.27% | 93.45% | -0.18%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: rn/Teacher/src/modules/graphql-speed-grader/components/SimilarityScore.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+}
+
+export type Props = StateProps
+
+const TURN_IT_IN_COLORS = {
+  PASS: '#008EE2',
+  DECENT: '#00AC18',
+  SKETCHY: '#FFC100',
+  DANGER: '#FC5E13',
+  THREAT_LEVEL_MIDNIGHT: '#EE0612',
```
Comment (armstrongnate) [inline rn/Teacher/src/modules/graphql-speed-grader/components/SimilarityScore.js]: @vanwagonet removed these when he did dark mode. I think we probably want to keep with what he did. What do you think?

https://github.com/instructure/canvas-ios/blob/55979cec1be09cbe75e8884bc2df556b5cf96f3c/rn/Teacher/src/modules/speedgrader/components/SimilarityScore.js#L74-L91

Comment (shichongrui) [reply rn/Teacher/src/modules/graphql-speed-grader/components/SimilarityScore.js]: oh good to know. I will change that.

**Comment by shichongrui** [comment]
Waiting on this canvas change ID to make it out to beta Id5f29018a481869b4c339a2c5ec98d9dd6e684c0

**Comment by armstrongnate** [comment]
I retriggered the Teacher build because it was failing to install from Bitrise for some reason.

**Pull request merged**